src/main/java/org/fungover/haze/HazeDatabase.java

```diff
@@ -17,6 +18,15 @@ public HazeDatabase() {
         this.lock = new ReentrantLock();
     }
 
+    public String getValue(String key) {
```

Comment:
Do we really need to make changes to HazeDatabase? The methods needed for LInsert must already exist since we have multiple other list Commands implemented already? This getValue must be availabel already?